packages/plugins/i18n/admin/src/components/CMHeaderActions.tsx

```diff
@@ -50,6 +50,59 @@ import type { I18nBaseQuery } from '../types';
  * LocalePickerAction
  * -----------------------------------------------------------------------------------------------*/
 
+interface LocaleOptionProps {
+  isDraftAndPublishEnabled: boolean;
+  locale: Locale;
+  status: 'draft' | 'published' | 'modified';
+  entryExists: boolean;
+}
+
+const LocaleOption = ({
+  isDraftAndPublishEnabled,
+  locale,
+  status,
+  entryExists,
+}: LocaleOptionProps) => {
+  const { formatMessage } = useIntl();
+
+  if (!entryExists) {
+    return formatMessage(
+      {
+        id: getTranslation('CMEditViewLocalePicker.locale.create'),
+        defaultMessage: 'Create <bold>{locale}</bold> locale',
+      },
+      {
+        bold: (locale: React.ReactNode) => <b>{locale}</b>,
+        locale: locale.name,
+      }
+    );
+  }
+
+  return (
+    <Flex width="100%" gap={1} justifyContent="space-between">
+      <Typography>{locale.name}</Typography>
+      {isDraftAndPublishEnabled ? (
+        <Status
+          display="flex"
+          paddingLeft="6px"
+          paddingRight="6px"
+          paddingTop="2px"
+          paddingBottom="2px"
```

Comment:
is it not possible to use the design system values here (1, 2, 3 etc) instead of absolute values?

Reply:
Couldn't make it work without big changes on DS and I think we need to revisit how Select component from DS is working currently 🤔

Reply:
let's not bother if it's complex 👍